app/audit/AuditEmploymentExpensesData.scala

```diff
@@ -0,0 +1,43 @@
+/*
```

Comment:
This PR seems almost entirely a copy of 1668
I cannot see the point of this PR as it all seem entirely implemented n SASS-1667

Suggest this PR is closed and any change I may have missed added to 1667

All 1667 comments apply here but should be addressed in 1667